hal/properties.c

```diff
@@ -147,6 +147,57 @@ static int hdlr_invalid (const char* data, char* ret) {
 	return RETURN_ERROR_SET_PROP;
 }
 
+static int hdlr_tx_a_rf_dac_dither_en (const char* data, char* ret) {
+	int r;
+	int en;
+	char buf[] = "mcu -c a -l \0";
+
+	r = sscanf( data, "%d", &en );
```

Comment:
It's a bit cleaner if you just do: if (sscanf( data, "%d", &en ) != 1)
Unless you have a strong reason against it.